src/components/Apps/UserConfirmDialog/index.vue

```diff
@@ -204,8 +204,8 @@ export default {
     logout() {
       window.location.href = `${process.env.VUE_APP_LOGOUT_PATH}?next=${this.$route.fullPath}`
     },
-    sendSMSCode() {
-      this.$axios.post(`/api/v1/authentication/mfa/select/`, { type: 'sms' }).then(res => {
+    sendCode() {
+      this.$axios.post(`/api/v1/authentication/mfa/select/`, { type: this.subTypeSelected }).then(res => {
         this.$message.success(this.$tc('VerificationCodeSent'))
         let time = 60
         this.smsBtnDisabled = true
```

Comment:
The code you provided has some inconsistencies and minor issues:

1. In "logout", `window.location.href` should be a relative path instead of absolute path, e.g., `/app/home`.
2. The logic inside each button click event handler is missing. For example: `<button @click="...">...</button>` needs to complete the event binding before its click handler.

For improvements:
1. Make sure all events bind properly.
2. Check for more detailed documentation and official examples if needed (e.g., Vuetify, Vue Router API).